documentation/SA1140.md

```diff
@@ -0,0 +1,57 @@
+## SA1140
+
+<table>
+<tr>
+  <td>TypeName</td>
+  <td>SA1140MaximumLineLength</td>
+</tr>
+<tr>
+  <td>CheckId</td>
+  <td>SA1140</td>
+</tr>
+<tr>
+  <td>Category</td>
+  <td>Readability Rules</td>
+</tr>
+</table>
+
+:memo: This rule is new for StyleCop Analyzers, it was added in 1.1.0
+
+## Cause
+
+A maximum number of characters in one line can lead to a better readability on screen and printed code.
+
+## Rule description
+
+This rule is disabled by default. To enable add the setting for maximum line length to your stylecop.json.
+
+```json
+{
+  "settings": {
+    "readabilityRules": {
+      "maximumLineLength": 100
+	}
+  }
+}
+```
+
+A value of 0 (default) for `maximumLineLength` disables the rule.
+
+## Known issues
+
+Tabs in source file are not expanded to spaces for evaluation of the line length. Only use this rule if spaces are used for indention.
```

Comment:
Should we do something about this? We could count the number of tabs in the line and use the configured tab width for that

Reply:
At the time this was first implemented, tab width might not have been included in stylecop.json. It's now available there so the feature could be implemented in this rule too.